install/terraform/modules/helm/helm.tf

```diff
@@ -153,6 +153,25 @@ resource "helm_release" "agones" {
     name  = "agones.featureGates"
     value = var.feature_gates
   }
+  
+    name  = "agones.featureGates"
+    value = var.feature_gates
+  }
```

Comment:
Hello @comerford I mean remove all lines which are duplicate with `featureGates`
```suggestion
```
Sorry, I was not able to make multiline suggestion from the first attempt
https://github.blog/changelog/2020-02-26-multi-line-code-suggestions-beta/